app/controllers/api/v1/financial_transactions_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Api::V1::FinancialTransactionsController < Api::V1::BaseController
+  include Concerns::CollectionScope
+
+  before_action :require_ordergroup
+
+  def index
+    render_collection search_scope
+  end
+
+  def show
+    render json: scope.find(params.require(:id))
+  end
+
+  private
+
+  def max_per_page
```

Comment:
why do we need this here? imho this is nothing specific to financial transactions

Reply:
The first financial transactions implementation returned all the user's transactions, I guess that's why I took over the option to be able to return _all_ the user's transactions, if needed.

I think it makes sense to limit this, as there is no built-in limit to the number of transactions and it keeps growing over time. Will remove.